frontend/src/pages/forgot_password.tsx

```diff
@@ -0,0 +1,133 @@
+      })
+      .catch((error: string) => {
+        setFirebaseError(error);
+      });
+  };
+  const { isMobile } = useWindowSize();
+
+  return (
+    <main className="flex h-screen w-full items-center justify-center">
+      <div className="flex h-full w-full items-center justify-center">
+        <div
+          className={cn(
+            "flex h-full flex-col",
+            isMobile ? "mt-[20%] w-[80%]" : "mb-[8%] w-[65%] justify-center",
+          )}
+        >
+          {isMobile && (
+            <div className="flex flex-col justify-center">
+              <div className="flex h-full flex-col justify-center">
+                <div className="flex flex-col items-center">
+                  <Image
+                    alt="company logo"
+                    src="/sidebar/logo.png"
+                    width={130}
+                    height={130}
+                    className=""
+                  />
+                </div>
+                <div className="mb-5 mt-5 flex flex-col items-center">
+                  <h1 className="font-[alternate-gothic] text-3xl max-lg:text-3xl">
+                    Forgot Password?
+                  </h1>
+                </div>
+              </div>
+            </div>
+          )}
+          {!isMobile && (
+            <div>
+              <h1 className="mb-7 font-[alternate-gothic] text-5xl text-black max-lg:text-5xl">
+                Forgot Password?
+              </h1>
+            </div>
+          )}
+
+          <div className="grid gap-5">
+            <form
+              onSubmit={handleSubmit(onSubmit)}
+              className="flex w-full flex-col justify-between gap-8"
+            >
+              <div>
+                <h1 className="text-lg font-light text-black text-pia_accent max-lg:text-lg">
+                  Email Address
+                </h1>
+                <Textfield
+                  register={register}
+                  name={"email"}
+                  label={""}
+                  type="email"
+                  placeholder="name@email.com"
+                  registerOptions={{ required: "Email cannot be empty" }}
+                />
+                {errors.email && (
+                  <h1 className="mt-1 flex items-center text-sm font-light text-orange-700">
+                    <AlertCircle className="mr-1 text-sm" />{" "}
+                    {typeof errors.email.message === "string" ? errors.email.message : null}
+                  </h1>
+                )}
+              </div>
+              <Button
+                type="submit"
+                className="rounded-md bg-pia_dark_green px-5 py-3 text-white"
+                disabled={resetSent}
+              >
+                Submit
+              </Button>
+              {resetSent && (
+                <h1 className="mt-1 flex items-center text-sm font-light text-green-700">
+                  <CheckCircle2 className="mr-1 text-sm" /> A reset password has been sent to your
```

Comment:
This should only popup if the reset email was sent properly. If I were to try to request a password for an account that we don't have an email for then we need some sort of message for that.

Reply:
Though I would like to note when I tried using an email that isn't associated with an account it never sent the password reset link which is good.

Reply:
I assumed that this was a sort of security measure, so that someone couldn't just figure out what emails do and don't have an account

Reply:
There are people that take this approach, where it says something like "A reset password was sent if your email was found in the database"

Reply:
That is a good point. I think the message you suggested will work.